commands/ssh.go

```diff
@@ -70,14 +64,18 @@ func RunSSH(ns string, out io.Writer) error {
 		return errSSHInvalidOptions
 	}
 
+	// CoreOS has no root user
```

Comment:
the SSH command seems to be all about making it easier to SSH into a droplet, so in that vein I think special casing for CoreOS makes ~sense~ tho I hate the precedent.